README.md

```diff
@@ -18,8 +18,8 @@ This client supports the following Google Cloud Platform services at a [General
 
 This client supports the following Google Cloud Platform services at a [Beta](#versioning) quality level:
 
-* [Cloud Firestore](#cloud-firestore-beta) (Beta)
-* [Cloud Natural Language](#cloud-natural-language-beta) (Beta)
+* [Cloud Firestore](http://github.com/googleapis/nodejs-firestore/) (Beta)
```

Comment:
Do you want to remove their snippets from the README as well? https://github.com/GoogleCloudPlatform/google-cloud-node#cloud-firestore-beta and https://github.com/GoogleCloudPlatform/google-cloud-node#cloud-natural-language-beta

Reply:
Not yet, although if they do not work, we should update or remove them.